### ISSUE TITLE
SvelteKit fails with process is not defined

### ISSUE THREAD

**Issue by linuxuser586** [pubkey/broadcast-channel#703]
## Case
Bug with proposed fix

## Issue
Using broadcast-channel with SvelteKit causes `process is not defined` error.

## Info
  - Environment: (browser)
  - Method: (Native/IndexedDB/Localstorage)
  - Stack: (Typescript, SvelteKit)

## Code

```js
  import { BroadcastChannel } from 'broadcast-channel';
  const channel = new BroadcastChannel('foobar');
  channel.postMessage('I am not alone');
  /* ... */
```

Please see https://github.com/linuxuser586/broadcast-channel-svelte-kit-bug for an example. The proposed fix is available at https://github.com/linuxuser586/broadcast-channel/tree/svelte-kit-fixes. Building with SvelteKit in dev and production with node adapter works with the fix. All tests pass with the fix.

**Comment by linuxuser586** [comment]
Fixed in latest release with small package.json change.

```json
"exports": {
    ".": {
      "node": {
        "import": "./dist/esnode/index.js",
        "default": "./dist/es5node/index.js"
      },
      "default": {
        "import": "./dist/esbrowser/index.js",
        "default": "./dist/lib/index.es5.js"
      }
    },
    "./package.json": "./package.json"
  },
```

**Comment by linuxuser586** [comment]
@pubkey It turns out this only works in development mode. As soon as I do a production build, I get import errors due to package.json defaulting to CommonJS and I don't want to update all my imports for CommonJS compatibility. I tried changing the esbrowser to .mjs as I did in my forked version and that created new problems. I'm going to continue using my forked version as that is the only version I can get to work in both development and production builds with no errors.
